build.sbt

```diff
@@ -98,3 +100,7 @@ lazy val `play26-core` = coreProject(Play_2_6).settings(
 lazy val `play27-core` = coreProject(Play_2_7).settings(
   suppressSemVerCheckOfNewScalaVersionsUntilNextVersion
 )
+lazy val `play28-core` = coreProject(Play_2_8).settings(
+  suppressSemVerCheckOfNewScalaVersionsUntilNextVersion,
+  libraryDependencies ++= Seq(playTest(Play_2_8))
```

Comment:
Q: Why do you have to do this extra step? I don't see it in any of the other project definitions.

Reply:
The others are only referencing play-test in their own tests but the play 2.8 implementation is also referencing it in the generated test helper